### PR TITLE
Update license for consistency

### PR DESCRIPTION
**Pull request by dhmlau**
### Description
- Update the license to make "IBM copyright" instead of "StrongLoop copyright" for being consistent with the other modules
- update `package.json` to the right value for `slt copyright` command to run properly
- run `slt copyright` to update the copyright header of source code

Related to: https://github.com/strongloop-internal/scrum-apex/issues/382

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -9,13 +13,13 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
...
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
```
Comment (virkt25) [inline LICENSE.md]: Need a new line here

File: test/test-caching.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-// Copyright IBM Corp. 2016,2017. All Rights Reserved.
-// Node module: loopback
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
```
Comment (shimks) [inline test/test-caching.js]: 2016,2018

Comment (dhmlau) [reply test/test-caching.js]: Actually i'm not sure why the tool changed it to `2017`. but can change it back to 2016.

File: LICENSE.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
-The MIT License (MIT)
+Copyright (c) IBM Corp. 2012,2017. All Rights Reserved.
```
Comment (shimks) [inline LICENSE.md]: 2012,2018

File: test/fixtures/echo-service.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
-// Copyright IBM Corp. 2016,2017. All Rights Reserved.
-// Node module: loopback
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
```
Comment (shimks) [inline test/fixtures/echo-service.js]: 2016,2018

**Comment by dhmlau** [comment]
Once it's ready to merge, I'll squash the commits first.

**Pull request merged**